lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2519,12 +2617,28 @@ def __init__(self, **kwargs):
         if self.module.params.get("healthcheck") and not healthcheck_supported:
             self.fail("docker or docker-py version is %s. Minimum version required is 2.0 to set healthcheck option." % (docker_version,))
 
+        found_device_limit_param = False
```

Comment:
This part will become much simpler with #47711 :) Whatever is merged first, the other will have to adjust accordingly ;)

Reply:
Yup. Minimum required version checks would become simpler when #47711 get merged.